src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/IActiveEditorContextTracker.cs

```diff
@@ -22,6 +22,11 @@ namespace Microsoft.VisualStudio.ProjectSystem.LanguageServices
     [ProjectSystemContract(ProjectSystemContractScope.UnconfiguredProject, ProjectSystemContractProvider.Private, Cardinality = ImportCardinality.ExactlyOne)]
     internal interface IActiveEditorContextTracker
     {
+        /// <summary>
+        ///     Gets or sets the current shared item context intellisense project name. Can be null.
+        /// </summary>
+        string? ActiveIntellisenseProjectContext { get; set; }
```

Comment:
This was added since it is needed for the non-VS specific implementation. We pass this through to the VS-specific interface.

Reply:
> Do you know if this is a full path to the project on disk?

Based on the error message, it is likely just some kind of ID (like a GUID). I didn't run/debug a build with these changes.

> And how does it related to the `contextId` parameter pass to `IsActiveEditorContext`?

They are one in the same. The class stores a collection of context IDs, and `IsActiveEditorContext` checks to see if the `contextId` passed in equals the one stored in `ActiveIntellisenseProjectContext`.